packages/apollo-server-codegen/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../tsconfig.base",
+  "compilerOptions": {
+    "rootDir": "./src",
+    "outDir": "./lib"
+  },
+  "include": ["src/**/*"],
+  "exclude": ["**/__tests__/*"]
```

Comment:
I believe you'll need a `references` key here for the `apollo-graphql` dependency.
`"references": [{ "path": "../apollo-graphql" }]`

Reply:
added!